docs/user-guide/release-notes/master.rst

```diff
@@ -15,6 +15,8 @@ New Features
   store rather than the certificate authority trust store bundled with
   ``python-requests``.
 
+* Content applicability for an updated repository is calculated in parallel.
```

Comment:
The API change needs to be called out as well. It's minor and probably doesn't affect many people, but we do need to fess up. :)